django/db/models/constraints.py

```diff
@@ -18,18 +18,23 @@
 
 class BaseConstraint:
     default_violation_error_message = _("Constraint “%(name)s” is violated.")
+    violation_error_code = None
     violation_error_message = None
 
     # RemovedInDjango60Warning: When the deprecation ends, replace with:
-    # def __init__(self, *, name, violation_error_message=None):
-    def __init__(self, *args, name=None, violation_error_message=None):
+    # __init__(self, *, name, violation_error_code=None, violation_error_message=None):
+    def __init__(
+        self, *args, name=None, violation_error_code=None, violation_error_message=None
+    ):
```

Comment:
We should add `violation_error_code` to the `zip(args, ["name", "violation_error_message"])`.

Reply:
Well, `violation_error_code` has never been a positional argument, so why should it be handled that way ?

Reply:
Right :+1: